app/controllers/users_controller.rb

```diff
@@ -42,8 +38,9 @@ def create
   def update
     respond_to do |format|
       if @user.update(user_params)
```

Comment:
- [ ] [Should](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_params.png) find the user by id from user_params